css.md

```diff
@@ -2,3 +2,19 @@
 
 We follow https://cssguidelin.es. In the css naming convention, we are aligned with the Hyphen Delimited convention instead of BEM-like one.
 For SASS, we follow https://sass-guidelin.es/
+
+## CSS Units
+
+Refer to this table to decide which unit to use in each case.
+
+| Unit       | Use for                                                          |
+|:-----------|:-----------------------------------------------------------------|
+| rem        | font-size, dimensions (margin, padding, height)                  |
```

Comment:
I agree that font-size could be only in rem but other properties as (margin padding, etc) could be in em, px.
Furthermore is good to know that rem is using based on the font-size to the root element.